controllers/storeController.js

```diff
@@ -82,7 +110,7 @@ const updateStore = async (req, res) => {
 
         if (!store) {
             return res.status(404).json({
-                status: "error",
```

Comment:
Ini pake new ApiError juga ya bang, contohnya dibawah
![image](https://github.com/akbarrahmatm/travel-be-project-team-5/assets/88664674/4f4f17bc-ea0e-4c86-a73c-59cbb03f7c50)